src/extension/vectortiles/src/main/java/org/geotools/renderer/lite/VectorMapRenderUtils.java

```diff
@@ -105,10 +110,99 @@ public static Query getStyleQuery(Layer layer, WMSMapContent mapContent) throws
         query.setProperties(Query.ALL_PROPERTIES);
 
         Hints hints = query.getHints();
+
         hints.put(Hints.FEATURE_2D, Boolean.TRUE);
 
+        // update the screenmaps
+        try {
```

Comment:
The screenmap is already set at the PipelineBuilder level, should probably be using that one (a code path to make it available needs to be created), but if you do, then the PreProcess step needs have the screenamap usage disabled. 
Even if they are kept separate, the two screenmaps should at least have the exact same config.